community-operators/vault/vaultoperator.v0.4.10.clusterserviceversion.yaml

```diff
@@ -0,0 +1,269 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: vault.banzaicloud.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: vaultoperator.v0.4.10
+  namespace: placeholder
+  annotations:
+    capabilities: Full Lifecycle
+    categories: "Security"
+    alm-examples: |
+      [
+        {
+          "apiVersion": vault.banzaicloud.com/v1alpha1",
+          "kind": "Vault",
+          "metadata": {
+            "name": "vault"
+          },
+          "spec": {
+            "size": 2,
+            "image": "vault:1.0.0"
+            "bankVaultsImage": "banzaicloud/bank-vaults:0.4.10"
+
+            # Support for custom Vault (and sidecar) pod annotations
+            "annotations": {
+              "prometheus.io/scrape": "true",
+              "prometheus.io/port": "9102"
+            }
+
+            # Describe where you would like to store the Vault unseal keys and root token.
+            "unsealConfig": {
+              "kubernetes": {
+                "secretNamespace": "default"
+              }
+            }
+
+            {
+              "config": {
+                "storage": {
+                  "file": {
+                    "path": "/vault/file"
+                  }
+                },
+                "listener": {
+                  "tcp": {
+                    "address": "0.0.0.0:8200",
+                    "tls_cert_file": "/vault/tls/server.crt",
+                    "tls_key_file": "/vault/tls/server.key"
+                  }
+                },
+                "telemetry": {
+                  "statsd_address": "localhost:9125"
+                },
+                "ui": true
+              }
+            }
+          }
+        }
+      ]
+    description: Creates and maintain highly-available Vault clusters on Kubernetes
+spec:
+  displayName: vault
```

Comment:
nit: The display name will be displayed in the UI as a user-friendly name for the operator. In the description below, "Vault Operator" is used with title case; this could be a potential candidate along with just "Vault"